app/views/admin/organizations/index.html.erb

```diff
@@ -36,7 +36,7 @@
                 <div class="col-3">
                   <%= render_filterrific_spinner %>
                 </div>
-                <div clas="col-9">
+                <div class="col-9">
```

Comment:
This shifts the button from being aligned right on the whole page (which it should be) to being above the 4th column.  Not desirable (though easily fixed, I'm sure.)

Reply:
Fixed.